modules/ROOT/pages/breakpoints-concepts.adoc

```diff
@@ -3,20 +3,73 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can configure breakpoints according to three parameters set in the Mule Breakpoints View.
+You can add breakpoints to your message processors so that the Studio debugger stops the execution of your flow at that particular processor. You can later inspect the content of the message as it exists at that point in the flow.
 
-[%header,cols="2*"]
-|===
-|Type |Description
-|Default (no parameters) |Default configuration for a user-defined breakpoint. The breakpoint is set on a user-defined event processor, and activated by default. When a message reaches the breakpoint event processor, Studio stops flow execution and displays the contents of the message in the Mule Debugger View.
-|Conditional |The breakpoint is activated only if a user-defined condition evaluates to _true_.
-// REVIEW Exception to Error breakpoint
-|Exception |This is a global parameter. When set, Studio stops flow execution _at any event processor_ where an exception occurs. The Mule Debugger View will display the message contents at the event processor that threw the exception.
-|===
+A breakpoint stops the execution of the flow when the message reaches it. You can also configure a breakpoint to execute when a specific condition in the message is met, or whenever an error occurs. See the types of breakpoints below for more details.
 
-When you set a breakpoint, Studio activates the breakpoint by default. The Mule Breakpoints View indicates the activation with a checked box to the left of the breakpoint name. Uncheck the box to deactivate a breakpoint. +
-Though deactivated, Studio does not delete the breakpoint. In the canvas, the breakpoint's corresponding event processor displays a gray circle instead of red.
+== Default Breakpoints
 
-Additionally you can activate the Mute Breakpoints option in the Mule Debugger View, to temporarily disallow the debugger to stop at your selected breakpoints.
+When you add a breakpoint, Studio enables the breakpoint by default.
 
+To add a default breakpoint, right-click a message processor and select *Add Breakpoint*:
+
+image::add-breakpoint.png[]
+
+To disable the breakpoint, right-click the message processor again and select *Disable Breakpoint*:
+
+image::disable-breakpoint.png[]
+
+Though disabled, Studio does not delete the breakpoint. In the canvas, the breakpoint's corresponding event processor displays a gray circle instead of a red circle.
```

Comment:
```suggestion
Although the breakpoint is disabled, Studio does not delete it. In the canvas, the breakpoint's corresponding event processor displays a gray circle instead of a red circle:
```